engine/src/main/java/com/datatorrent/stram/debug/StdOutErrLog.java

```diff
@@ -52,6 +54,7 @@ public static void tieSystemOutAndErrToLog()
       logger.warn("found appender {} instead of RollingFileAppender", appender);
     }
 
+    LoggerUtil.addAppenders();
```

Comment:
Currently, there is an assumption that all appenders are known at a container start. Please see static initialization of LoggerUtil. The new functionality breaks that assumption, so it is necessary to fix how fileAppender is set in LoggerUtil.